data/mods/CRT_EXPANSION/monsters/crt_monster.json

```diff
@@ -2,7 +2,7 @@
   {
     "id": "mon_slasher",
     "type": "MONSTER",
-    "name": "Slasher Necromorph",
+    "name": "Slasher Bio-Recombinant",
```

Comment:
The Lore component could go into master, as could the file rename (I would think). This would appear to be a string change?

Reply:
Yeah this needs String freeze. Thanks!